_docs/for-contributors/2017-01-01-building-from-source.md

```diff
@@ -39,7 +47,31 @@ Switch to the latest release branch, for example, `cardano-sl-1.0`:
 
     $ git checkout cardano-sl-1.0
 
-Then enter `nix-shell`:
+## Pure Nix build mode (recommended)
```

Comment:
Remove Pure (it adds no value, but scared people)

Reply:
Done.